Snippets/NHibernate/NHibernate_8/Usage.cs

```diff
@@ -18,7 +18,6 @@ void Simple(EndpointConfiguration endpointConfiguration)
         endpointConfiguration.UsePersistence<NHibernatePersistence, StorageType.Subscriptions>();
         endpointConfiguration.UsePersistence<NHibernatePersistence, StorageType.Timeouts>();
         endpointConfiguration.UsePersistence<NHibernatePersistence, StorageType.Outbox>();
-        endpointConfiguration.UsePersistence<NHibernatePersistence, StorageType.GatewayDeduplication>();
```

Comment:
Here, where we're just being comprehensive, we can just remove the Gateway mention.